feat: Add new repository-level Coverage pages COV-256 (#1689)
codacy/docs: docs/repositories-coverage/commits.md

-   Committer, SHA hash, and parent commits
-   Date
-   Link to the commit on your Git provider

![Commit information](images/commits-detail-information.png)

<!--coverage-overview-start-->
## {{ page.meta.page_name.capitalize() }} coverage overview {: id="coverage-overview"}

This area displays the coverage gate status and an overview of the coverage metrics for the {{ page.meta.page_name }}:

-   The quality gate status is either **Up to coverage standards** or **Not up to coverage standards** depending on the [coverage gate rules](../repositories-configure/adjusting-quality-gates.md) for your repository.

    If you don't have any rules enabled for {{ page.meta.page_name }}s, the status is always **Up to coverage standards**.

-   The following coverage metrics for the {{ page.meta.page_name }}, displayed either as a **positive or negative variation**, {% if page.meta.page_name == "commit" %}or **no variation** (represented by `=`){% else %}**no variation** (represented by `=`), or **not applicable** (represented by `∅`){% endif %}:

{% if page.meta.page_name == "commit" %}
    -   **Coverage variation:** Variation of code coverage percentage relative to the parent commit

>     -   **Total coverage:** Coverage value for the current commit

Reviewer: I’m wondering if this is clear and easy to understand. When we say that Total Coverage is the coverage for the current commit, will some users think that this is the Coverage for the changes in the commit?

Author: Good question. While I was creating the page I noticed that this is a new metric that we're displaying on the commit details page ([it's not there on the Codacy Quality page](https://docs.codacy.com/repositories/commits/#quality-overview)).

"Intuitively" I assumed that it was the total coverage for the repository on that commit (since a commit is a snapshot of all files on the repository). If this metric was the coverage of just the changes introduced by the commit, it should be called "diff coverage" - also to keep things consistent between commits and pull requests. :thinking: 

Anyway, on the docs side we can already clarify this better. Here's a proposal:

```suggestion
    -   **Total coverage:** Coverage value of the repository at this commit
```
